features/audit_resources.feature

```diff
@@ -0,0 +1,36 @@
+Feature: audit with additional resources
+
+  Scenario: with one additional resource
+    Given I successfully run `conjur variable create $ns_foo bar`
+
+    When I evaluate the expression
```

Comment:
Those aren't very good steps. The code here looks like it should go in a step definition instead.